table/block_based_table_builder.cc

```diff
@@ -592,14 +592,14 @@ void BlockBasedTableBuilder::Flush() {
   if (!ok()) return;
   if (r->data_block.empty()) return;
   WriteBlock(&r->data_block, &r->pending_handle);
-  if (ok()) {
+  if (ok() && !r->table_options.skip_table_builder_flush) {
     r->status = r->file->Flush();
   }
   if (r->filter_block != nullptr) {
     r->filter_block->StartBlock(r->offset);
   }
   r->props.data_size = r->offset;
-  ++r->props.num_data_blocks;
+  ++r->props.num_data_blocks; 
```

Comment:
Can you please remove the space? this make the linter complain.
If you are using arcanist, you can run "arc lint"